source/overview/license-and-subscription.rst

```diff
@@ -109,6 +109,71 @@ Is my license available immediately?
+
+How do I renew my license?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You can renew your license in Mattermost via the blue renewal reminder banner. Select Renew Now to begin the process. You can also visit https://mattermost.com/renew, and complete the form provided. 
+
+How long does it take to renew a license?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Once you’ve started the renewal process, we will be in contact with you to confirm your order and send you the order form. There may be additional paperwork required. Once we have the signed order form and (if applicable) the necessary paperwork from you, we are able to process the renewal and issue your license key within 24 hours.  
+
+
+What happens to my trial/subscription if I don't renew in time?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+From version 5.24, if you don't renew within the 60-day renewal period, a 10 day grace period is provided. During this period your Mattermost installation runs as normal, with full access to Enterprise features. 
+
+When the grace period expires, your Enterprise version is downgraded to Team Edition. Enterprise features are disabled.
+ 
+What happens when my trial/subscription expires?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you don't renew your license within the 10-day grace period, your Mattermost version is automatically downgraded to Team Edition so you can still access and use Mattermost. However, Enterprise features will no longer be available and if you are currently using them, the functionality will no longer be accessible. 
+
+When you renew, the Enterprise features will become available with the previous configuration (provided no action such as user migration has been taken). 
+
+Which features are affected when my Enterprise license expires?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you are using the following Enterprise features, they will be impacted 
+
+.. csv-table::
+    :header: "Feature", "How it's affected", "What steps do I need to take?"
+
+    "Elasticsearch", "Elasticsearch is automatically disabled and will start using the default database for indexing posts", "None needed."
+    "AD/LDAP, SAML SSO, Office 365 SSO, and Google SSO login options removed from the sign-in page.", "- Users who previously signed in with one of these methods are no longer able to. 
+    - Users who were already signed in can continue to use Mattermost until their session expires or until they log out.", "Users must be migrated to email authentication using the `CLI <https://docs.mattermost.com/administration/command-line-tools.html#mattermost-user-migrate-auth>`_. This can also be done via **System Console > Users**. Select the drop-down menu for the relevant member, choose **Switch to Email/Password**, enter a new password, and choose **Reset**."
```

Comment:
Looks like the CLI doesn't support migrating "to" email. So the CLI option isn't actually viable.